### PR TITLE
Fixes #19620 - restores compatibility with Foreman 1.16

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by dschlenk** [comment]
+1

File: app/lib/actions/foreman_chef/host/destroy.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def resource_locks
 
         def plan(host)
           action_subject(host)
-          if (::Setting::ForemanChef.auto_deletion && proxy = host.chef_proxy)
+          if (::Setting::ForemanChef['auto_deletion'] && proxy = host.chef_proxy)
```
Comment (akofink) [inline app/lib/actions/foreman_chef/host/destroy.rb]: Why not be consistent with strings vs. symbols? I'd prefer symbols.

**Comment by akofink** [comment]
@ares what's going on with this PR? It's quite stale, and this issue is being hit by users on 1.16.

**Comment by ares** [comment]
I'm sorry I plan to look in all PRs on foreman_chef and do the release at the end of the next week. I need to do more testing and I lost my setup so it will take some time.

**Comment by ares** [comment]
Thanks @akofink @dschlenk, replaced by https://github.com/theforeman/foreman_chef/pull/79

**Pull request closed without merging**